packages/sil-governance/src/routes/create-proposal/components/FormOptions/ReleaseGuaranteeFunds.tsx

```diff
@@ -2,21 +2,46 @@ import { FormApi } from "final-form";
 import { Block, TextFieldForm, Typography } from "medulas-react-components";
 import React from "react";
 
+const AMOUNT_LABEL = "Amount";
+export const RELEASE_QUANTITY_FIELD = "Quantity";
+const RELEASE_QUANTITY_PLACEHOLDER = "Quantity";
+export const RELEASE_DIGITS_FIELD = "Fractional Digits";
+const RELEASE_DIGITS_PLACEHOLDER = "Fractional Digits";
+export const RELEASE_TICKER_FIELD = "Token Ticker";
+const RELEASE_TICKER_PLACEHOLDER = "Token Ticker";
+
 interface Props {
   readonly form: FormApi;
 }
 
 const ReleaseGuaranteeFunds = ({ form }: Props): JSX.Element => {
   return (
-    <Block width="100%" display="flex" alignItems="center" marginTop={2} marginBottom={2}>
-      <Typography>Custom Field for ReleaseGuaranteeFunds </Typography>
-      <TextFieldForm
-        name="ReleaseGuaranteeFunds"
-        form={form}
-        placeholder="ReleaseGuaranteeFunds"
-        fullWidth
-        margin="none"
-      />
+    <Block marginTop={2} display="flex" alignItems="center">
+      <Typography>{AMOUNT_LABEL}</Typography>
+      <Block marginLeft={2}>
+        <TextFieldForm
+          name={RELEASE_QUANTITY_FIELD}
+          form={form}
+          placeholder={RELEASE_QUANTITY_PLACEHOLDER}
+          margin="none"
+        />
+      </Block>
+      <Block marginLeft={2}>
+        <TextFieldForm
+          name={RELEASE_DIGITS_FIELD}
+          form={form}
+          placeholder={RELEASE_DIGITS_PLACEHOLDER}
+          margin="none"
+        />
+      </Block>
+      <Block marginLeft={2}>
+        <TextFieldForm
+          name={RELEASE_TICKER_FIELD}
+          form={form}
+          placeholder={RELEASE_TICKER_PLACEHOLDER}
+          margin="none"
+        />
+      </Block>
```

Comment:
Those fields can be hardcoded to "9" and "IOV", such that the user does not need to type-in the values anymore